index.html

```diff
@@ -251,7 +251,7 @@ <h2 class="about-me-header">About me</h2>
               </ul>
               </ul>
               <ul class="skil">
-              <li class="about-skill-lang"><img src="images/circle.png" alt="Round-dot">&nbsp;&nbsp;Skills</li>
+              <li class="about-skill-lang"><img src="images/circle.png" alt="Circle">&nbsp;&nbsp;Skills</li>
```

Comment:
- [ ] If the image is not important for understanding the content — for example, it is just decoration or "eye candy" — it should have null alt (alt=""). One way to help determine if an image should have a null alt is to ask yourself: If the image was removed, would the user still get all the information from the page? Kindly fix it.